staticd/static_zebra.c

```diff
@@ -43,6 +43,8 @@
 #include "static_nht.h"
 #include "static_vty.h"
 
+bool debug;
+
```

Comment:
Is this a typo or left-over from development?

Reply:
intentionally left in, the code tests for it and uses it.  I modeled it after the debug booleans in lib in a couple of places.  The intent is to add actual debug's into staticd in the near future.  If you need to turn on the debugs that are in there now, you modify it via a debugger.